### ISSUE TITLE
Feature request: Track specific URL instead of domain

### ISSUE THREAD

**Issue by alexbash7** [towfiqi/serpbear#119]
It would be very helpful if the script could track particular URLs for some keywords instead of only domain. This will help in the situation when few links from the same domain exists in the search results. For example for keyword search "trading app reddit" Google return multiple links from Reddit, but I need track position only of one particular

**Comment by s3w47m88** [comment]
I agree. I may have my programming staff contribute a patch for this if the author will accept it.

**Comment by towfiqi** [comment]
@s3w47m88 It's already on the to-do list. Hopefully, will start working on it in the next few days.

**Comment by s3w47m88** [comment]
Thank you @towfiqi  - I will stand by then. But let me know if you'd like us to assist.

**Comment by towfiqi** [comment]
Thanks for offering to help.
